lib/scss/theme/_variables.scss

```diff
@@ -4,10 +4,12 @@ $theme-highlight-color: #e6522c;
 // Headings
 $headings-color: $theme-highlight-color;
```

Comment:
I think we have to keep `$headings-color` black as per what we had previously on this line:
https://github.com/miniohq/gluegun/commit/b40dc632a897ee5e3342b0db17e8440c389ee848#diff-a999414d8e61572fa44ec8a01ad1c82bR6

Reply:
I will send the fix for this for us to deploy it today

Reply:
I couldn't push directly into your branch, would appreciate if you are able to make this change @rushenn

Reply:
Please check now. I've fixed it.